Atomic.md

```diff
@@ -0,0 +1,618 @@
+  },
+  {
+    TransactionType: "OfferCreate",
+    Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+    TakerGets: "6000000",
+    TakerPays: {
+      currency: "GKO",
+      issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+      value: "2"
+    },
+    AtomicTxn: {
+      Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+      OuterSequence: 3,
+      BatchIndex: 0
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  },
+  {
+    TransactionType: "Payment",
+    Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+    Destination: "rDEXfrontEnd23E44wKL3S6dj9FaXv",
+    Amount: "1000",
+    AtomicTxn: {
+      Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+      OuterSequence: 3,
+      BatchIndex: 1
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  }
+]
+```
+</details>
+
+### 6.2. Multiple Accounts
+
+In this example, two users are atomically swapping their tokens, XRP for GKO.
+
+#### 6.2.1. Sample Transaction
+
+<details open>
+<summary>
+
+The inner transactions are still not signed, but the `AtomicSigners` field is needed on the outer transaction, since there are two accounts' inner transactions in this `Atomic` transaction.
+</summary>
+
+```typescript
+{
+  TransactionType: "Atomic",
+  Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+  Flags: "1",
+  TxnIDs: [
+    "A2986564A970E2B206DC8CA22F54BB8D73585527864A4484A5B0C577B6F13C95",
+    "0C4316F7E7D909E11BB7DBE0EB897788835519E9950AE8E32F5182468361FE7E"
+  ],
+  RawTransactions: [
+    {
+      RawTransaction: {
+        TransactionType: "Payment",
+        Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        Destination: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+        Amount: "6000000",
+        AtomicTxn: {
+          Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+          OuterSequence: 4,
+          Sequence: 10,
+          BatchIndex: 0
+        },
+        Sequence: 0,
+        Fee: "0",
+        SigningPubKey: "",
+        TxnSignature: ""
+      }
+    },
+    {
+      RawTransaction: {
+        TransactionType: "Payment",
+        Account: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+        Destination: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        Amount: {
+          currency: "GKO",
+          issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+          value: "2"
+        },
+        AtomicTxn: {
+          Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+          OuterSequence: 4,
+          Sequence: 20,
+          BatchIndex: 1
+        },
+        Sequence: 0,
+        Fee: "0",
+        SigningPubKey: "",
+        TxnSignature: ""
+      }
+    }
+  ],
+  AtomicSigners: [
+    {
+      AtomicSigner: {
+        Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        SigningPubKey: "03072BBE5F93D4906FC31A690A2C269F2B9A56D60DA9C2C6C0D88FB51B644C6F94",
+        Signature: "304502210083DF12FA60E2E743643889195DC42C10F62F0DE0A362330C32BBEC4D3881EECD022010579A01E052C4E587E70E5601D2F3846984DB9B16B9EBA05BAD7B51F912B899"
+      }
```

Comment:
As I asked about above, having an `AtomicSigner` for the outer transaction, when it's also signing the entire transaction seems like a waste of ledger space.

Reply:
I'm okay with making that change.